### PR TITLE
[Omega] Now brig apc and HoP apc are connected to power grid

### LINKED ISSUE #28957: [Omega] APC in security and HoP office aren't connected to the power grid
The entire HoP office isn't connected and the APC in the brig doesn't have a cable conected to his terminal

### PR DESCRIPTION
**Pull request by PromethTK11**
Fixes #28957

### CODE REVIEW

**Comment by ShizCalev** [comment]
those cables don't actually have their directions set. 

they also have dirty edits.

**Review by Cyberboss — CHANGES_REQUESTED** [review]
You only changed the icon_state you need to change the dir as well

**Comment by PromethTK11** [comment]
@Cyberboss Done

Btw, it's odd, literally every cable of pubbystation and omegastation doesn't have `d1` and `d2`, they just change in `icon_state`

**Comment by ShizCalev** [comment]
@PromethTK11 update your branch. i fixed it a while ago.

File: _maps/map_files/OmegaStation/OmegaStation.dmm [hunk 1]
```diff
@@ -6944,6 +6955,13 @@
 	on = 1;
 	scrub_Toxins = 0
 	},
+/obj/structure/cable{
+	d1 = 4;
+	d2 = 8;
+	icon_state = "4-8";
+	pixel_x = 0;
```
Comment (ShizCalev) [inline _maps/map_files/OmegaStation/OmegaStation.dmm]: dirty var

File: _maps/map_files/OmegaStation/OmegaStation.dmm [hunk 2]
```diff
@@ -6944,6 +6955,13 @@
...
+	tag = ""
```
Comment (ShizCalev) [inline _maps/map_files/OmegaStation/OmegaStation.dmm]: dirty var

File: _maps/map_files/OmegaStation/OmegaStation.dmm [hunk 3]
```diff
@@ -3282,6 +3282,13 @@
 /obj/effect/turf_decal/stripes/line{
 	dir = 8
 	},
+/obj/structure/cable{
+	d1 = 4;
+	d2 = 8;
+	icon_state = "4-8";
+	pixel_x = 0;
+	tag = ""
```
Comment (ShizCalev) [inline _maps/map_files/OmegaStation/OmegaStation.dmm]: dirty var

File: _maps/map_files/OmegaStation/OmegaStation.dmm [hunk 4]
```diff
@@ -3282,6 +3282,13 @@
 /obj/effect/turf_decal/stripes/line{
 	dir = 8
 	},
+/obj/structure/cable{
+	d1 = 4;
+	d2 = 8;
+	icon_state = "4-8";
+	pixel_x = 0;
```
Comment (ShizCalev) [inline _maps/map_files/OmegaStation/OmegaStation.dmm]: dirty var

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Remove the dirty vars

**Comment by PromethTK11** [comment]
@Cyberboss Done
The conflicts can be resolved with the web editor?

**Comment by Cyberboss** [comment]
Gonna go with @ShizCalev's

**Pull request closed without merging**